examples/plotting/solar_cycle_example.py

```diff
@@ -19,6 +20,8 @@
 noaa_predict = ts.TimeSeries(NOAAPREDICT_TIMESERIES, source='noaapredictindices')
 
```

Comment:
You need to edit lines 17 and 18 to have the following text:

```
# For this example we will use the sunpy sample data which has old (2015) sunspot record and prediction from
# the  Space Weather Prediction Center (SWPC) part of the National Oceanic and Atmospheric Administration (NOAA).
```

Reply:
done.